src/patternfly/components/Label/styles.scss

```diff
@@ -0,0 +1,48 @@
+@import "../../patternfly-utilities";
+
+.pf-c-label {
+  // Component
+  --pf-c-label--PaddingTop: var(--pf-global--spacer--xs);
+  --pf-c-label--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-label--PaddingBottom: var(--pf-global--spacer--xs);
+  --pf-c-label--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-label--LineHeight: var(--pf-global--LineHeight--md);
+  --pf-c-label--BorderRadius: 2px;
+
+  // Color
+  --pf-c-label--BackgroundColor: var(--pf-global--primary-color--100);
+  --pf-c-label--Color: var(--pf-global--Color--light-100);
+
+  // Font
+  --pf-c-label--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-label--FontWeight: var(--pf-global--FontWeight--semi-bold);
```

Comment:
looks like this should be regular from the design

Reply:
thanks :)